### DISCUSSION TITLE
Waiting list for Addons

### DISCUSSION THREAD

**Discussion by niklasberlin** [pretix/pretix#1967] [Ideas]
At our event we offer an Show on satturday, but we have limited seating capacity in the venue. Therefor we added the show-ticket as an addon so that only participants of our four day event can buy a ticket for the show. Right now there is no possibility for our participants to get on a waiting list for a show ticket when they are currently sold out -> inconvienient and not totaly fair because the participant need to check the shop on a regular basis to maybe get a ticket which got canceled.

Workaround could be to have bundeld products but right now we have 4 Tickets with a total of 6 Variations, if we would need to create another set of bundled products it gets confusing in the Shop (to many possibilitys)

A Waiting list for Addons would solve this, so people could add the addon later to their existing order when the addon gets availabe again.

**Comment by raphaelm** [comment]
Absolutely. However, the "adding to their existing order" later on is not as easy as it sounds. We do have a major overhaul of the waiting list on our roadmap, which will hopefully include this, but we haven't scheduled it yet.
